src/rev_ai/language_identification_client.py

```diff
@@ -20,17 +20,20 @@ class LanguageIdentificationClient(GenericApiClient):
     # Default api name of Rev AI language identification api
     api_name = 'languageid'
 
-    def __init__(self, access_token):
+    def __init__(self, access_token, url=None):
         """Constructor
 
         :param access_token: access token which authorizes all requests and links them to your
                              account. Generated on the settings page of your account dashboard
                              on Rev AI.
+        :param url (optional): url of the Rev AI API deployment to use, defaults to the US
+                    deployement, i.e. 'https://api.rev.ai', which can be referenced as
+                    RevAiApiDeploymentConfigMap[RevAiApiDeployment.US]['base_url'].
         """
 
         GenericApiClient.__init__(self, access_token, self.api_name, self.api_version,
                                   LanguageIdentificationJob.from_json,
-                                  LanguageIdentificationResult.from_json)
+                                  LanguageIdentificationResult.from_json, url)
```

Comment:
i'm not sure i understand the pattern used for whether a parameter on a new line or not, are you just fitting as many in a line as possible? this is fine, i personally i think have each parameter on a new line but again i don't think this is important either way

Reply:
i think the pattern is fitting the most in a line